File: C/Listas/Lista 01/ex11.c
//Calcule e mostre a área de um triângulo (área é igual a (base x altura) dividido por 2)
#include <stdio.h>

int main(void){
    int base, altura;
    float area;
    printf("Digite a base de seu triângulo: ");
    scanf("%d", &base);
    printf("Insira a altura do seu triângulo: ");
    scanf("%d", &altura);
    area=(base*altura)/2;
    
    printf("A área do seu triângulo com base nos valores informados é igual a %.1fm²", area);
}

File: Java/extra/Lista 01/src/ex11.java
//Calcule e mostre a área de um triângulo (área é igual a (base x altura) dividido por 2)

import java.util.Scanner;

public class ex11 {
    static double area(int a, int b) {
        double x = ((double) a * (double) b) / 2;
        return x;
    }

    public static void main(String[] args) {
        int base, altura;
        System.out.println("Informe o valor da base do triângulo:");
        try (Scanner leitor = new Scanner(System.in)) {
            base = leitor.nextInt();
            System.out.println("\nDigite o valor da altura do triângulo:");
            altura = leitor.nextInt();
        }
        System.out.format("\nA área do triângulo = %.1f", area(altura, base));
    }
}
